pplanner_settings:
  grid_map_topic: "/elevation_mapping/elevation_map_raw"             #"/elevation_mapping/elevation_map_raw"
  grid_cell_topic: "/free_grid_cells"
  grid_cells_of_frontier_topic_: "/frontier_grid_cells"
  grid_cells_of_bound_topic_: "/bound_grid_cells"
  tra_map_sizex: 4.0
  tra_map_sizey: 4.0
  tra_map_resolution: 0.04 #0.03
  edges_of_one_point_max: 10
  pplaner_map_resolution: 0.15
  traversability_score_threshold_local_graph: -0.6
  traversability_score_threshold_local_graph_for_frontier: 0.3
  traversability_score_threshold_global_graph: 0.50
  traversability_score_threshold_low_bound: 0.5
  bad_point_threshold: 3 #15
  worst_point_threshold: 2 #5
  if_bad_point_threshold_improve_automatically: false
  point_threshold: 5 #20
  point_threshold_frontier: 3
  limit_box_z: 1.35
  limit_box_z_for_edge: 0.25
  if_grid_cells_pub: true
  spinner_thread: 10
  grid_cell_frame: "world_planner" #world
  elevation_layer: "elevation"
  traversability_layer: "traversability"
  step_layer: "step"
  traversability_supplementary_layer: "step"
  variance_layer: "variance"
  pplanner_map_publish_fps: 10
  iterator_map_fps: 1
  if_iterator_map_with_map_Callback: false
  map_of_frontier_update_fps: 10
  if_edge_pub: true
  if_vertex_marker_also_pub: true
  submap_of_frontier_update_fps: 10
  map_of_bound_update_fps: 10
  world_frame: "world_planner" #world
  track_frame: "base_link_planner" #base_link
  attractive_gain: 0.0
  repulsive_gain: 0.01
  bound_radiu_: 0.6
  if_plus_: true
  x_plus_: 3.0
  y_plus_: 3.0
  z_plus_: 0.3
  Max_plus_count: 5
  Orientation_of_tracker_update_fps: 10
  Safety_circumvent_of_tracker_update_fps: 10
  lookahead_time_for_safety_circumvent_: 1.0
  x_forward_offset_for_safety_ckeck_: 0.52
  x_backward_offset_for_safety_ckeck_: 0.52
  robot_width: 0.51
  Local_Planner_fps: 1
  Update_Local_Graph_Manager_fps: 1
  PathTrackerStateCheckFps: 1
  bad_point_threshold_for_check_safety: 5
  local_attractive_gain: 0.0
  local_repulsive_gain: 0.02
  local_bound_radiu_: 0.8
  VisulaizeRobotPathPassedFps: 5.0
  VisualizePointsIssuedFps: 5.0
  ExplorerFps: 0.5
  UpdateTargetFps: 1.0
  UpdateExplorerGraphManagerFps: 2.0
  LambdaSize: 1000.11
  LambdaDistance: 0.101666666
  LambdaObstacle: 0.0101
  LambdaZDistance: 6.1
  IfUsePathDistanceInsteadOfEuclideanDistance: true
  IfAvoidObstaclesByChangeVelocity: false
  HandleEmergencyByTurnAroundFps: 1.0
  TimesOfGlobalTargetIdRepetitionThreshold: 2
  PplannerRoughGlobalMapResolution: 0.5
pplanner_tracker_settings:
  if_tracker_enable: true
  path_track_fps: 10
  cmd_topic_: "/cmd_vel"  #"/smb_velocity_controller/cmd_vel"
  goal_reaching_threshold: 0.9
  pose_update_fps: 10
  World_Frame: "world_planner" #world
  Robot_Frame: "base_link_planner" #base_link
  look_ahead_distance: 0.5
  look_ahead_error_margin: 0.2
  carrot_point_topic: "/carrot_point"
  curvature_scaling_angle: 0.52
  critical_angle: 0.52
  max_ang_vel: 0.5
  desired_vel: 0.5
  num_point_in_path_look_ahead_for_turn_: 5
  min_inver_point: 5

  